requirements.txt

```diff
@@ -17,6 +17,7 @@ chardet>=3.0,<3.1  # https://pypi.python.org/pypi/chardet
 babel>=2.6,<2.7  # http://babel.pocoo.org/en/latest/changelog.html
 pyxdg>=0.26,<0.27  # https://freedesktop.org/wiki/Software/pyxdg/
 Jinja2>=2.10,<2.11  # http://jinja.pocoo.org/docs/latest/changelog/
+tomli
```

Comment:
This should probably come with a version.

Reply:
This can now be removed as we upgraded to Python 3.12 which (since 3.11) supports TOML natively: https://docs.python.org/3/library/tomllib.html